### PR TITLE
Log deprecated flag usage

### PR DESCRIPTION
**Pull request by antekresic**
## Description

This PR contains a couple of changes that are connected to the renaming process of old flags. Main feature is the addition of logging usage of deprecated flags with recommendations on how to migrate them.

I'm unsure if any of this should be added to the changelog, perhaps only the change to remove the deprecated flags in 0.11.0 version. I would like some input on that before adding it.

## Merge requirements

Please take into account the following non-code changes that you may need to make with your PR:

- [ ] CHANGELOG entry for user-facing changes
- [ ] Updated the relevant documentation

### CODE REVIEW

File: pkg/log/log.go [hunk 1]
```diff
@@ -43,6 +43,17 @@ func ParseFlags(fs *flag.FlagSet, cfg *Config) *Config {
 	return cfg
 }
 
+// InitDefault is used to start the logger with sane defaults before the we can configure it.
```
Comment (JamesGuthrie) [inline pkg/log/log.go:46]:
```suggestion
// InitDefault is used to start the logger with sane defaults before we can configure it.
```

**Review by JamesGuthrie — APPROVED** [review]
Looks good, one minor nit.

File: pkg/log/log.go (hunk 1 above)
Comment (antekresic) [reply pkg/log/log.go:46]: :+1:

File: pkg/runner/flags.go [hunk 2]
```diff
@@ -225,3 +246,52 @@ func validate(cfg *Config) error {
...
+			}
+
+			if !set {
+				panic(fmt.Sprintf("trying to set an flag alias for a flag that is missing: %s", f.Name))
+			}
+		}
+	})
+}
+
+func flagAliasesUsed(fs *flag.FlagSet) map[string]string {
```
Comment (Harkishen-Singh) [inline pkg/runner/flags.go:268]:
```suggestion
func deprecatedFlagsUsed(fs *flag.FlagSet) map[string]string {
```
Is this better?

File: pkg/runner/flags.go [hunk 3]
```diff
@@ -225,3 +246,52 @@ func validate(cfg *Config) error {
...
+
+			switch {
+			case f.Name == "migrate":
+				// Migrate flag is a special case since it's not just a rename
+				// of an old flag. We handle it separately when logging
+				// use of deprecated flags.
+				return
+			case err != nil:
+				fallthrough
+			case i != 1:
```
Comment (Harkishen-Singh) [inline pkg/runner/flags.go:287]: shouldn't this be `i < 1` as more than one deprecated flags can be passed?

File: pkg/util/util.go [hunk 4]
```diff
@@ -12,8 +12,7 @@ import (
 )
 
 const (
-	PromNamespace     = "promscale"
-	aliasDescTemplate = "Alias for -%s flag. "
+	PromNamespace = "promscale"
 )
```
Comment (Harkishen-Singh) [inline pkg/util/util.go:16]: Nit

```
const PromNamespace = "promscale"
```

**Review by Harkishen-Singh — APPROVED** [review]
A question and minor nits

File: pkg/runner/flags.go (hunk 3 above)
Comment (antekresic) [reply pkg/runner/flags.go:287]: Actually no, an alias can have only one flag that it is aliasing.

**Pull request merged**